### PR TITLE
Slack wording

### PR DESCRIPTION
**Pull request by spolu**
Fixes https://github.com/dust-tt/tasks/issues/47

Screenshot from 2023-10-18 14-58-10
Screenshot from 2023-10-18 14-58-15

### CODE REVIEW

**Review by gabhubert — COMMENTED** [review]
suggested edits

File: front/pages/w/[wId]/builder/data-sources/[name]/index.tsx [hunk 1]
```diff
@@ -323,7 +323,8 @@ function StandardDataSourceView({
 const CONNECTOR_TYPE_TO_HELPER_TEXT: Record<ConnectorProvider, string> = {
   notion: "Explore the Notion pages and databases Dust has access to:",
   google_drive: "Google Drive folders and files Dust has access to:",
-  slack: "Slack channels Dust has access to:",
+  slack:
+    "Synchronize Slack channels by inviting @Dust (the Slack app that was connected to your Slack workspace) to join those channels. Slack channels Dust has access to:",
```
[the remark this answers is not in the document]
Comment (spolu) [reply front/pages/w/[wId]/builder/data-sources/[name]/index.tsx:327]: Not quite right. These are the channels we are invited to and that were selected for synchronization

File: front/components/ConnectorPermissionsModal.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const CONNECTOR_TYPE_TO_RESOURCE_LIST_TITLE_TEXT: Record<
   string | null
 > = {
   slack:
-    "Dust is currently invited to the channels below. Select if data from those channels should be synchronized or not.",
+    "@Dust (the Slack app that was connected to your Slack workspace) was invited, and therefore has access, to the channels below. Select which channels you want Dust to synchronize as part of the Slack managed data source:",
```
[the remark this answers is not in the document]
Comment (spolu) [reply front/components/ConnectorPermissionsModal.tsx]: udpated

File: front/pages/w/[wId]/builder/data-sources/[name]/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (spolu) [reply front/pages/w/[wId]/builder/data-sources/[name]/index.tsx]: Pushed an update keeping the standard wording we have for the second sentence

[the remark this answers is not in the document]
Comment (spolu) [reply front/pages/w/[wId]/builder/data-sources/[name]/index.tsx]: ... with more substance

Screenshot from 2023-10-18 17-47-31

**Review by gabhubert — APPROVED** [review]
discussed in the issue

**Pull request merged**